python/ray/parameter.py

```diff
@@ -313,3 +362,10 @@ def _check_usage(self):
         if numpy_major <= 1 and numpy_minor < 16:
             logger.warning("Using ray with numpy < 1.16.0 will result in slow "
                            "serialization. Upgrade numpy if using with ray.")
+
+    def _wrap_port(self, port):
+        # 0 port means select a random port for the grpc server.
+        if port is None or port == 0:
+            return []
+        else:
+            return [port]
```

Comment:
nit - define this inline with the `update_pre_selected_port` function?

Reply:
Good idea. Done